src/main/resources/db/migration/V1_12__service_user.sql

```diff
@@ -0,0 +1,23 @@
+create table service_user (
+    referral_id uuid not null unique,
+    address text,
+    crn text,
```

Comment:
Can we treat CRN as a required (and lookup) key for users, please? We shouldn't have any without a CRN identifier (there are processes in court admin which guarantee this as far as I know)

Reply:
yes

are there any such rules on nomis or PNC numbers? or are they sort of "bonus" identifiers?

Reply:
i've just taken CRN out of this and used it as the primary service user ID in the referral table.

Reply:
or do you want to be able to query all referral_service_user_data entities with a certain crn?

Reply:
perhaps if we have a requirement for this in the future we can add it then. we have all the data.